src/components/StorageOverview/Events/Events.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+  DashboardCardTitleHelp,
+} from '../../Dashboard/DashboardCard';
+import { StorageOverviewContextGenericConsumer } from '../StorageOverviewContext';
+
+export const Events = ({ Component }) => (
+  <DashboardCard>
+    <DashboardCardHeader>
+      <DashboardCardTitle>Events</DashboardCardTitle>
+      <DashboardCardTitleHelp>help for events</DashboardCardTitleHelp>
+    </DashboardCardHeader>
+    <DashboardCardBody id="events-body" className="kubevirt-events__body">
```

Comment:
please move body to `Dashboard` folder (it would contain `div` with `id` and `className` of DashboardCardBody) so it can be reused here and in cluster overview.

Reply:
@rawagner Ahh missed your comment and did it differently. Directly used Cardbody and passed just the ID. please check if that's ok otherwise we do it your way.